README.md

```diff
@@ -277,6 +277,38 @@ sys.addShutdownHook(system.terminate())
 ```
 
 ## Changelog  
+
+### 3.0.1 (2017-10-27)
+Changes since v3.0.0
+  - Akka 2.5 release only, please use the 2.7.x release for Akka 2.4 
+  - Changed oracle test-docker image
+  - Align JournalTables with the sql schemas (issue #130)
+  - Current events by tag fix (issue #124)
+  - fixes for max ordering jump (issue #126)
+  - Allow configuration of ask timeout when querying maxOrdering
```

Comment:
You can mention that this is issue #123

Reply:
yep. I was looking for it, but somehow couldn't find it